backend/app/api/endpoints/devices.py

```
@@ -735,6 +735,7 @@ async def start_device_terminal(
 @router.post("/{device_id}/code-server", response_model=DeviceSessionResponse)
 async def start_device_code_server(
     device_id: str,
+    payload: DeviceSessionCreate | None = Body(default=None),
```

Ruff (0.15.21)

[warning] Do not perform function call `Body` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)